### ISSUE TITLE
Need Mocks for Testing

### ISSUE THREAD

**Issue by mbielski** [ngx-translate/core#669]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => check the FAQ and search github for a similar issue or PR before submitting
[ ] support request => check the FAQ and search github for a similar issue before submitting
[X] feature request
```

**Current behavior**
All of the actual modules are being loaded when components that contain them are put to test. This leads to longer compile and execution time. As an example, my current project unit tests ran in under 1.2 seconds before I added this project to mine. My unit tests now take 4.5 seconds to run. TS compile time is also noticeably longer.

**Expected/desired behavior**
Unless you're actually testing your translations you don't need to load the full functionality and/or the language files. What is needed are mocks of everything that don't reach out to load the language files.

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar. You can use this template as a starting point: http://plnkr.co/edit/tpl:WccVZSBM0rUgq2sXSUbe
Take a look at the way that [Ionic-Mocks](https://www.npmjs.com/package/ionic-mocks) is done. These should probably be like that.

**What is the expected behavior?**
I expect that the mocks will satisfy the TS builds and not drastically increase the test times.

**What is the motivation / use case for changing the behavior?**
Faster TS compiling and faster test runs.

**Please tell us about your environment:**
Windows 10 Home 64-bit
i7-4900MQ @ 2.8ghz
32G RAM
Node 8.x

* **ngx-translate version:** 8.0.x

* **Angular version:** 4.3x
 
* **Browser:** 
Ionic 3

**Comment by pvilardo** [comment]
Yes, absolutely necessary, no need to test translations in my components.

**Comment by mwootendev** [comment]
I created the ngx-translate-testing (NPM: https://www.npmjs.com/package/ngx-translate-testing) module to help ease testing. See if that can help you with your testing setup.
